Algorithm.CSharp/BasicTemplateOptionsFrameworkAlgorithm.cs

```diff
@@ -160,7 +160,7 @@ public override IEnumerable<IPortfolioTarget> CreateTargets(QCAlgorithm algorith
             {"Information Ratio", "0"},
             {"Tracking Error", "0"},
             {"Treynor Ratio", "0"},
-            {"Total Fees", "$4.00"},
+            {"Total Fees", "$3.00"},
             {"Fitness Score", "0"},
```

Comment:
Mind adding the "why" for these changes to the PR? Does this mean there's no order fee for the assignment?

Reply:
So Option exercise was NOT happening in master, but a `MarketOnClose` order was being added since the incoming split event for AAPL, so in PR option was exercised at 9:33 with no fee